### PR TITLE
Implemented Savegame

### PR DESCRIPTION
**Pull request by Jan130**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -114,5 +114,8 @@ Thumbs.db
 /ios/IOSLauncher.app
 /ios/IOSLauncher.app.dSYM
 
-#all lock files
+# all lock files
 *.lock
+
+# Savegame files
+/assets/GameData/*
```
Comment (itsSanoi) [inline .gitignore:121]: I have no idea from that shit but i think its ok xD

**Review by itsSanoi — APPROVED** [review]
I think the rest is fine

**Review by Xinoi — CHANGES_REQUESTED** [review]
there is a merge conflict. Everything else looks fine

**Comment by Jan130** [comment]
Fixed Merge conflict

**Review by Xinoi — APPROVED** [review]
ok you can merge it

**Comment by Xinoi** [comment]
> Fixed Merge conflict

what was the problem bdw ?

**Comment by Jan130** [comment]
I think I removed an empty line or something

**Pull request merged**